gulpfile.js

```diff
@@ -98,7 +123,8 @@ const javascripts = {
       ]
     }).then(bundle => {
       return bundle.write({
-        file: paths.dist + 'javascripts/sharing-button.js',
+        dir: paths.dist + 'javascripts/',
+        entryFileNames: '[name]-[hash].js',
```

Comment:
Minor: could we use a loop to generate this hash for each of the 3 input files? At the moment we have a lot of duplication (each one entry is the same).

Something like this:

```
const javascripts = Object.fromEntries(
  ['sharing-button-init.mjs', ...].map((name) => {
    [name, () => {
      rollup.rollup(...
```

Reply:
I’ve done something similar to your suggestion in aadec78

Reply:
Nice 👍